src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
```

Comment:
The class 'NumberBall' should have an access modifier to define its visibility explicitly. If the class is intended to be used within the package only, you can leave it as default; otherwise, consider adding 'public'.